server/src-lib/Hasura/App.hs

```diff
@@ -166,7 +167,11 @@ initialiseCtx hgeCmd rci = do
   eDbId <- liftIO $ runExceptT $ Q.runTx pool (Q.Serializable, Nothing) getDbId
   dbId <- either printErrJExit return eDbId
 
-  return (InitCtx httpManager instanceId dbId loggers connInfo pool, initTime)
+  -- get the pg version
```

Comment:
You can use [`serverVersion`](https://github.com/hasura/pg-client-hs/blob/master/src/Database/PG/Query/Transaction.hs#L251) to get the underlying Postgres version.

There seems to be a lot of `runExceptT $ runTx` followed by `printErrJExit`, so might as well clean up this section by having a function as follows:
```haskell
-- | runs a transaction, on error, it is logged and the program is aborted
runTxIO :: Q.Pool -> Q.IsolationLevel -> Tx e a -> IO a
runTxIO = ...
```